_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -2,35 +2,41 @@
 permalink: "/tutorials/IOSXR-Ansible"
 author: Mike Korshunov
 excerpt: Getting started with IOSXR and Ansible playbooks
-published: true
+published: false
 title: "IOS-XR Ansible"
 date: "2016-05-16 01:18 +0530"
 ---
+
+{% include toc icon="table" title="IOS-XR Ansible" %}
+
+
+
 # Introduction
-In this tutorial we are going to cover XRv4 configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to XRv64.
+In this tutorial we are going to cover IOS-XRv configuration with Ansible.We will set up and try a very simple configuration with a linux machine connected to IOS-XRv.
+
 
 ## Prerequisites
 - Computer with with 4-5GB free RAM;
 - Vagrant;
 - Ansible;
-- [XRv64 image](http://engci-maven-master.cisco.com/artifactory/appdevci-snapshot/);
+- [IOS-XRv image](http://engci-maven-master.cisco.com/artifactory/appdevci-snapshot/);
+- [Vagrantfile and scripts for provisioning](https://cisco.box.com/s/wykoquj1z76gbedq2mj0pgt0sfhwjvhv)
 
 ### Vagrant pre-setup
 
-To add XR box, image should be downloaded, after this issue the command:
-	
-    $ vagrant box add xrv64 iosxrv-fullk9-x64.box_2016-05-07-19-04-50
+Setup was tested on Windows, but it's the same for other OS. To add XR box, image should be downloaded, after this issue the command:
+  
+    $ vagrant box add xrv iosxrv-fullk9-x64.box_2016-05-07-19-04-50
 
 Image for Ubuntu will be downloaded from official source:
-	
+  
     $ vagrant box add ubuntu/trusty64
     
-Let's check for result, we should have box available, box ubuntu/trusty64 and xrv64 should be displayed:  
+Let's check for result, we should have box available, box ubuntu/trusty64 and xrv should be displayed:
 
-![Box validation](https://cisco.box.com/shared/static/rwa6hxojdjcwu1ln8h0zu5oq6ij30def.png)
-![Box validation](https://xrdocs.github.io/xrdocs-images/assets/images/xr_ansible_01_box_list-Copy.png)
+![Box validation](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xr-ansible-tutorial/xr_ansible_01_box_list.png)
 
-Vagrantfile with 2 Vagrant boxes should be like:
+Vagrantfile is in [Box](https://cisco.box.com/s/wykoquj1z76gbedq2mj0pgt0sfhwjvhv) it contain 2 Vagrant boxes and looks like:
```

Comment:
Don't put things on box. Create a public repository on github (under your username is fine) and publish things (especially code) there.